src/stories/image.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react"
+
+import { StoryProps } from "./storyProps"
+import StoryContainer from "./storyContainer"
+import Image, { ImageProps } from "../components/image"
+
+export default {
+  title: "Image",
+  component: Image,
+  args: {
+    storyName: "Image",
+  },
+}
+
+interface Props extends StoryProps<unknown>, ImageProps {}
+
+const Template: FC<Props> = (args) => {
+  return (
+    <StoryContainer
+      title={args.storyName}
+      component={<Image {...args}>{args.children}</Image>}
+    />
+  )
+}
+
+export const Default = Template.bind({})
```

Comment:
There are quite some more variations of the image. Can we showcase more here?

Reply:
We can but there will be no visible difference in Storybook

Reply:
`galleryImage` and `cover` should make a difference. you might need some additional markup around the image in the story (which is fair game) to demo the latter

Reply:
You're right ! With additional style to the StoryContainer we can clearly show `cover`, could not make a huge difference with` galleryImage` though (which is called `fullScreenHeight` now). I will try again tomorrow.